docs/api.md

```diff
@@ -56,6 +56,10 @@ see [`mozilla/fxa-js-client`](https://github.com/mozilla/fxa-js-client).
   * [Recovery codes](#recovery-codes)
     * [GET /recoveryCodes (:lock: sessionToken)](#get-recoverycodes)
     * [POST /session/verify/recoveryCode (:lock: sessionToken)](#post-sessionverifyrecoverycode)
+  * [Recovery keys](#recovery-keys)
+    * [POST /recoveryKeys (:lock: sessionToken)](#post-recoverykeys)
+    * [GET /recoveryKeys/{recoveryKeyId} (:lock: accountResetToken)](#get-recoverykeysrecoverykeyid)
+    * [POST /account/reset/recoveryKeys (:lock: accountResetToken)](#post-accountresetrecoverykeys)
```

Comment:
Naming nit: should this be "/account/reset/recoveryKey" singular, since it only consumes a single recovery key?